quipucords/quipucords/settings.py

```diff
@@ -36,6 +36,18 @@
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
 
+class RelativePathnameFormatter(logging.Formatter):
+    """Define a custom formatter to add a relative pathname."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def format(self, record):
+        """Add a relative_pathname attribute."""
+        record.relative_pathname = os.path.relpath(record.pathname, BASE_DIR)
```

Comment:
I'm wondering why ruff isn't complaining about `os.path` in use here 🤔 

Would you be so kind to replace this with the pathlib alternative

```suggestion
        record.relative_pathname = Path(record.pathname).relative_to(BASE_DIR)
```